os.cpp

```diff
@@ -236,9 +238,9 @@ unsigned short int readFile(unsigned short int start){
 // Convert a command as a string into an integer value to simplify
 // switch statements using commands.
 int cmdToInt(string cmd) {
-	if (strcasecmp(cmd.c_str(), "run")) return 0;
-	else if (strcasecmp(cmd.c_str(), "dmp")) return 1;
-	else if (strcasecmp(cmd.c_str(), "nop")) return 2;
-	else if (strcasecmp(cmd.c_str(), "stp")) return 3;
+	if (cmd == "run") return 0;
+	else if (cmd == "dmp") return 1;
+	else if (cmd == "nop") return 2;
+	else if (cmd == "stp") return 3;
```

Comment:
String comparison here needs to be case insensitive. I'll write a helper function so we can still do case insensitive comparison without `strcasecmp()`